src/empty-states/src/EmptyState.js

```diff
@@ -0,0 +1,184 @@
+      padding={majorScale(6)}
+      backgroundColor={backgroundColor}
+      justifyContent="center"
+    >
+      <Pane display="flex" alignItems="flex-start" flex={1} height="100%">
+        <Pane paddingRight={majorScale(6)}>
+          <Pane
+            display="flex"
+            justifyContent="center"
+            alignItems="center"
+            borderRadius="50%"
+            backgroundColor={imageBgColor}
+            width={majorScale(9)}
+            height={majorScale(9)}
+          >
+            {React.cloneElement(image, { size: majorScale(4) })}
+          </Pane>
+        </Pane>
+        <Pane display="flex" flexDirection="column" paddingRight={majorScale(6)}>
+          <Heading size={500}>{title}</Heading>
+          {description && (
+            <Paragraph color="muted" marginTop={majorScale(2)}>
+              {description}
+            </Paragraph>
+          )}
+          {hasFooter && (
+            <Pane marginTop={majorScale(5)} display="flex">
+              {primaryCta && React.cloneElement(primaryCta, { marginRight: majorScale(2) })}
+              {secondaryCta}
+              {(primaryCta || secondaryCta) &&
+                anchorCta &&
+                React.cloneElement(anchorCta, { marginLeft: majorScale(2) })}
+              {!primaryCta && !secondaryCta && anchorCta}
+            </Pane>
+          )}
+        </Pane>
+      </Pane>
+    </Pane>
+  )
+}
+
+/* eslint-enable react/prop-types */
+
+/* eslint-disable react/prop-types */
+
+const VerticalOrientation = ({ background, description, image, imageBgColor, primaryCta, title }) => {
+  const { colors } = useTheme()
+  const backgroundColor = background === 'light' ? 'white' : colors.gray75
+
+  return (
+    <Pane
+      display="flex"
+      alignItems="center"
+      flexDirection="column"
+      justifyContent="center"
+      flex={1}
+      backgroundColor={backgroundColor}
+      paddingX={majorScale(5)}
+      paddingY={majorScale(5)}
+      height="100%"
+      width="100%"
+    >
+      <Pane
+        display="flex"
+        justifyContent="center"
+        alignItems="center"
+        borderRadius="50%"
+        backgroundColor={imageBgColor}
+        width={majorScale(7)}
+        height={majorScale(7)}
+      >
+        {React.cloneElement(image, { size: majorScale(3) })}
+      </Pane>
+      <Heading marginTop={majorScale(2)} textAlign="center">
+        {title}
+      </Heading>
+      <Paragraph marginTop={majorScale(2)} textAlign="center">
+        {description}
+      </Paragraph>
+      {primaryCta && React.cloneElement(primaryCta, { marginTop: minorScale(5) })}
+    </Pane>
+  )
+}
+
+/* eslint-enable react/prop-types */
+
+const PrimaryButton = props => {
+  return <Button appearance="primary" {...props} />
+}
+
+const SecondaryButton = props => {
+  return <Button appearance="minimal" {...props} />
+}
+
+const LinkButton = props => {
+  return <Link {...props} target="_blank" size={300} lineHeight="34px" />
```

Comment:
The way this currently behaves - we're assuming links passed in would strictly be external links. Should we remove the explicit `target="_blank"`, and instead leave that up to consumers? I can see folks passing through single-page route changes here... i.e.


```
<EmptyState 
    // ... 
    anchorCta={<EmptyState.LinkButton is={ReactRouterLink} to="/path/in/my-app"}
/>
```